src/apiserver/json_helper.cc

```diff
@@ -200,6 +203,75 @@ JsonReader& JsonReader::operator&(std::string& s) {  // NOLINT
     return *this;
 }
 
+JsonReader& JsonReader::operator&(std::shared_ptr<openmldb::sdk::SQLRequestRow>& parameter) {  // NOLINT
+    size_t size;
+    this->StartArray(&size);
+    ::hybridse::vm::Schema schema;
+    std::vector<Value> values;
+    int32_t str_length = 0;
+    for (auto i = 0; i < size; i++) {
+        if (!error_) {
+            auto col = schema.Add();
+            if (CURRENT.IsBool()) {
+                col->set_type(::hybridse::type::kBool);
+                values.push_back(Value(CURRENT.GetBool()));
+            } else if (CURRENT.IsUint()) {
+                col->set_type(::hybridse::type::kInt64);
+                values.push_back(Value(CURRENT.GetUint()));
+            } else if (CURRENT.IsInt()) {
+                col->set_type(::hybridse::type::kInt64);
+                values.push_back(Value(CURRENT.GetInt()));
+            } else if (CURRENT.IsDouble()) {
+                col->set_type(::hybridse::type::kDouble);
+                values.push_back(Value(CURRENT.GetDouble()));
+            } else if (CURRENT.IsString()) {
+                col->set_type(::hybridse::type::kVarchar);
+                values.push_back(Value(CURRENT.GetString(), CURRENT.GetStringLength()));
+                str_length += CURRENT.GetStringLength();
+            } else {
+                error_ = true;
+            }
+            Next();
+        }
+    }
+    this->EndArray();
+
+    ::hybridse::sdk::SchemaImpl* schema_impl = new ::hybridse::sdk::SchemaImpl(schema);
+    std::shared_ptr<::hybridse::sdk::Schema> schema_shared(schema_impl);
+    std::set<std::string> _rs{};
+    parameter.reset(new openmldb::sdk::SQLRequestRow(schema_shared, _rs));
+
+    if (!parameter->Init(str_length)) goto fast_fail;
+    for (auto i = 0; i < size; i++) {
+        auto col = schema.Get(i);
+        auto& val = values.at(i);
+        bool ok = false;
+        switch (col.type()) {
+            case ::hybridse::type::kBool:
+                ok = parameter->AppendBool(val.GetBool());
+                break;
+            case ::hybridse::type::kInt64:
```

Comment:
these types are parsed from JSON ([qsliu2017/OpenMLDB@093e2ce/src/apiserver/json_helper.cc#L217](https://github.com/qsliu2017/OpenMLDB/blob/093e2ce560fb2a106ac7cbe3e2305e62fcdea4da/src/apiserver/json_helper.cc#L217)). since JSON only have several types, we only cover these types.